README.md

```diff
@@ -410,4 +426,221 @@ See documentation for [django-environment-config] for more details.
+
+#### Admin UI
+
+`apps/admin-ui/.env.local` contains the environment variables for the Admin UI.
+
+| Name                                | Description                                                   |
+| ------------------------------------| ------------------------------------------------------------- |
+| NEXT_PUBLIC_BASE_URL                | Application baseUrl in {host}/{baseUrl}. `/kasittely`         |
+| TILAVARAUS_API_URL                  | Tilavaraus backend hostname                                   |
+| RESERVATION_UNIT_PREVIEW_URL_PREFIX | Customer url to display the preview of a Reservation Unit     |
+| EMAIL_VARAAMO_EXT_LINK              | Feedback url link (has to be an url)                          |
+| SENTRY_DSN                          | Sentry dsn                                                    |
+| SENTRY_ENVIRONMENT                  | Sentry environment, for example 'test', 'prod'                |
+| SENTRY_AUTH_TOKEN                   | Auth token for sentry cli                                     |
+| SKIP_ENV_VALIDATION                 | `true` to not validate env configuration                      |
+
+#### Environment variables
+
+`TILAVARAUS_API_URL` is required to be set because the SSR (NodeJS) can't connect to the backend without it.
+In local development they are on different hosts (i.e. ports).
+In production the SSR server is separate from the backend server with both behind a reverse proxy.
+Even though the browser client is running on the same host as the backend, the SSR requires this URL to connect to the backend.
+
+`ENABLE_FETCH_HACK` fixes issues with localhost and IPv6 in Node 18+.
+It has no effect if `TILAVARAUS_API_URL` is not `localhost` but should NOT be enabled on server environments.
+
+`SKIP_ENV_VALIDATION` dont validate the `.env` before running. Should only be used for CI builds and tests.
+
+`PROFILE_UI_URL` used to display a link to Helsinki profile in the Customer UI.
+
+`RESERVATION_UNIT_PREVIEW_URL_PREFIX` used to link to Customer UI from Admin UI. Should be the full url to `reservation-unit` without the `pk`.
+ex. `https://tilavaraus.dev.hel.ninja/reservation-unit`. Changes on each environment.
+
+##### Sentry variables
+
+Don't enable these locally unless making changes to sentry.
+
+`SENTRY_DSN` used to connect to sentry project.
+
+`SENTRY_AUTH_TOKEN` used to upload source maps to sentry (CI build step).
+
+`SENTRY_ENVIRONMENT` used to set the environment in Sentry (tags the logs). For local testing use `local`.
+
+### Scripts
+
+All available top level scripts are listed in the root `package.json`. Most of them use turborepo to run
+the same command in all packages. They can be run with `pnpm {command}`.
+
+Target commands to specific packages using the `--filter` flag.
+``` shell
+# only that package
+pnpm {command} --filter {package_name}
+
+# only that package and it's dependencies
+pnpm {command} --filter {package_name}...
+```
```

Comment:
Explain why I would want to do this. Why not run commands for everything every time?